Configuration/StandardSequences/python/Digi_cff.py

```diff
@@ -42,3 +42,45 @@
     from FastSimulation.Configuration.DigiAliases_cff import loadDigiAliases
     loadDigiAliases(premix = False)
     from FastSimulation.Configuration.DigiAliases_cff import generalTracks,ecalPreshowerDigis,ecalDigis,hcalDigis,muonDTDigis,muonCSCDigis,muonRPCDigis
+
+#phase 2 common mods
+def _modifyDigitizerPhase2Common( theProcess ):
+    theProcess.load("CalibCalorimetry/HcalPlugins/Hcal_Conditions_forGlobalTag_cff")
+    theProcess.es_hardcode.toGet = cms.untracked.vstring(
```

Comment:
Since the module will not run if no code asks for its data product, I'm OK with always loading it even in other eras which do not need it. Althought, I'd suggest giving it a better label name to express what its job is.